metis-indexing/src/main/java/eu/europeana/indexing/tiers/view/LanguageBreakdown.java

```diff
@@ -0,0 +1,64 @@
+package eu.europeana.indexing.tiers.view;
+
+import java.util.List;
+
+public class LanguageBreakdown {
+
+  private int potentialLanguageQualifiedElements;
+  private int actualLanguageQualifiedElements;
+  private int actualLanguageQualifiedElementsPercentage;
```

Comment:
As a percentage wouldn't we want this as a double instead of int?

Reply:
Not sure how useful the decimal fractions would be in this case but I could update this to float, perhaps not double.